Source/Scene/Primitive.js

```diff
@@ -683,6 +683,30 @@ define([
         return renamedVS + '\n' + showMain;
     };
 
+    Primitive._appendDistanceDisplayConditionToShader = function(primitive, vertexShaderSource) {
+        if (!defined(primitive._attributeLocations.distanceDisplayCondition)) {
+            return vertexShaderSource;
+        }
+
+        var renamedVS = ShaderSource.replaceMain(vertexShaderSource, 'czm_non_distanceDisplayCondition_main');
+        var distanceDisplayConditionMain =
+            'attribute vec3 boundingSphereCenter3DHigh; \n' +
+            'attribute vec3 boundingSphereCenter3DLow; \n' +
+            'attribute float boundingSphereRadius; \n' +
+            'attribute vec2 distanceDisplayCondition; \n' +
```

Comment:
Combine with `boundingSphereCenter3DHigh` and `boundingSphereCenter3DLow` and the `w` component?